porcelain/miner.go

```diff
@@ -295,6 +298,57 @@ func MinerGetOwnerAddress(ctx context.Context, plumbing mgoaAPI, minerAddr addre
 	return address.NewFromBytes(res[0])
 }
 
+// MinerGetSectorSize queries for the sector size of the given miner.
+func MinerGetSectorSize(ctx context.Context, plumbing mgoaAPI, minerAddr address.Address) (*types.BytesAmount, error) {
```

Comment:
BLOCKING: This and `MinerGetLastCommittedSectorID` below need tests. At the very least we need a test that uses a fake `MessageQuery` that creates a response with a sector size and the real `ActorGetSignature`  to deserialize the value into a sector size.

Reply:
Done, although I feel a little icky having introduced such a test. 

Given that the method under test uses `abi.Deserialize`, I don't see how to test these two methods without re-implementing the ABI serializers (in test code). 

I get how the DI struct allows us to write fine-grained tests, but I'm not convinced that (in this case) the benefit (low-level test coverage) is worth the cost (increased coupling to implementation-level stuff).